enlighten/file_io/Configuration.py

```diff
@@ -177,14 +177,14 @@ def load_text(self):
 
     def parse(self):
         """ Load as ConfigParser object. """
-        self.config = configparser.ConfigParser(interpolation=None)
+        self.config = configparser.ConfigParser(interpolation=None, strict=False)
```

Comment:
This allows enlighten.ini files with duplicate keys to be loaded (don't crash)

Reply:
I'm liking the inline-github comments. This one should be added to the source file too.